login-workflow/src/new-architecture/screens/EulaScreen/EulaScreenBase.tsx

```diff
@@ -0,0 +1,76 @@
+/** eslint-ignore */
+import React, { useCallback, useState } from 'react';
+import { EulaScreenProps } from './types';
+import { WorkflowCard } from '../../components/WorkflowCard';
+import { WorkflowCardActions } from '../../components/WorkflowCard/WorkflowCardActions';
+import { WorkflowCardBody } from '../../components/WorkflowCard/WorkflowCardBody';
+import { WorkflowCardHeader } from '../../components/WorkflowCard/WorkflowCardHeader';
```

Comment:
You can simplified the imports like this
`import { WorkflowCard, WorkflowCardActions, WorkflowCardBody, WorkflowCardHeader } from '../../components/WorkflowCard';
`

Reply:
Done